Update linking.md (#96)
GeyserMC/GeyserWiki: _docs/floodgate/linking.md

Instructions and information about how to link can also be found here: https://link.geysermc.org/

Before we introduced Global Linking, you always had to link your Java and Bedrock account on every individual server you visited (that has Floodgate). Global Linking is here to fix that problem. Link once, join everywhere.<br>

> Keep in mind: While having your accounts linked, you will use the Java accounts' location, inventory data, and achievements etc. regardless of which platform you signs in from (therefore, "synchronising" the player data). The player data from the Bedrock account will not be accessible until you unlink again - therefore, you should transfer everything (enderchest contents, items, armor) to the Java account before linking to not "lose" your Bedrock progress. If you forgot to do this, you can unlink, transfer everything over, and link again. 

Reviewer: Java account's

Since it's a singular player we're talking about.

Also enderchest -> ender chest

Author: fixed, thanks for the correction

> Keep in mind: While having your accounts linked, you will use the Java account's location, inventory data, and achievements etc. regardless of which platform you signs in from (therefore, "synchronising" the player data). The player data from the Bedrock account will not be accessible until you unlink again - therefore, you should transfer everything (ender chest contents, items, armor) to the Java account before linking to not "lose" your Bedrock progress. If you forgot to do this, you can unlink, transfer everything over, and link again. 

Reviewer 2: ```suggestion
Keep in mind: While having your accounts linked, you will use the Java account's location, inventory data, and achievements etc. regardless of which platform you sign in from (therefore, "synchronising" the player data). The player data from the Bedrock account will not be accessible until you unlink again. As a result, you should transfer everything (ender chest contents, items, armor) to the Java account before linking to not "lose" your Bedrock progress. If you forgot to do this, you can unlink, transfer everything over, and link again.
```

4. Enter the random number on the other account by typing `/linkaccount <code>`
5. You should get kicked from the server on both your Bedrock and Java account with the message that it had successfully linked your accounts.

> To unlink your accounts, join the GlobalLinkServer (as described above for linking) on either Java or Bedrock, and use the /unlinkaccount command. 

Reviewer: and use the -> or use the

Author: But i'm trying to describe how globally unlink? I've added "To unlink your globally linked accounts[..]" at the beginning
Cuz you cannot unlink globally linked accounts on the website, and the "unlinkaccount" for global linking only exists on the GlobalLinkServer

Reviewer: Oh, I see now. I was thinking about the local `/unlinkaccount` command on Floodgate servers.

> To unlink your globally linked accounts, join the GlobalLinkServer (as described above for linking) on either Java or Bedrock, and use the /unlinkaccount command. 

Reviewer 2: ```suggestion
To unlink your globally linked accounts, join the GlobalLinkServer (as described above for linking) on either Java or Bedrock, and use the `/unlinkaccount` command. 
```

If you have selected `mysql` a new data folder for the database should be generated inside the Floodgate data folder. You can enter your database credentials in there. After you did that restart your server once more.

> That should be it. You can then link your accounts by following the instructions you get when typing `/linkaccount`. 

Reviewer 2: ```suggestion
That should be it. You can then link your accounts by following the instructions you get when typing `/linkaccount`.
```
